wao.io.siteoptimizationsubdomainstage.json

```diff
@@ -0,0 +1,21 @@
+{
+  "providerId": "wao.io",
+  "providerName": "wao.io",
+  "serviceId": "SiteOptimizationSubDomainStage",
+  "serviceName": "Site Optimization Sub Domain Stage",
+  "version": 1,
+  "description": "Enables wao.io site optimization for a sub domain.",
+  "multiInstance": true,
+  "hostRequired": true,
+  "syncBlock": false,
+  "warnPhishing": true,
```

Comment:
Not sure why you need warnPhising.  Nothing in this template is based on a variable, so you are not at risk for phising attacks.

Reply:
I will remove it then.